github.yaml

```diff
@@ -58,6 +58,31 @@ systems:
             - name: resource_path
               description: This field is used as the path portion of the API call url
 
+      createRepo:
+        target:
+          system: github
+          function: api
+        parameters:
+          URL: 'https://api.github.com/orgs/{{ .ctx.org }}/repos'
+          method: POST
+          content:
+            name: '{{ .ctx.name }}'
+            private: true
+        export:
+          repoid: '{{ .data.json.id | int | toString }}'
+
+      addRepoToInstallation:
```

Comment:
maybe we could do this for Renovate and semantic release commit checker bots too?